src/controllers/reviewController.ts

```diff
@@ -0,0 +1,82 @@
+import Review from '../models/Review';
+import { Request, Response, NextFunction } from 'express';
+import { NotFoundError } from '../errors/httpErrors';
+
+async function retrieveReview(req: Request, res: Response, next: NextFunction) {
```

Comment:
Perhaps we should retrieve the usernames along with the review so that it will be easier for the frontend?

Reply:
The User who made the review is already included in the query though

Reply:
I don't even think we have a username field 😆